CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to Roslyn
+
+Guidelines for contributing to the Roslyn repo.
+
+## Submitting Pull Requests
+
```

Comment:
:bulb: Before having a bunch of scary rules, start with a comment that says we're here to help and invites the user to ask questions if they don't understand anything.